server/events/vcs/client.go

```diff
@@ -25,6 +25,10 @@ type Client interface {
 	// relative to the repo root, e.g. parent/child/file.txt.
 	GetModifiedFiles(repo models.Repo, pull models.PullRequest) ([]string, error)
 	CreateComment(repo models.Repo, pullNum int, comment string) error
+	// HideOldComments will hide old comments left from previous plan runs to reduce
+	// clutter in a pull/merge request. This will not delete the comment, since the
```

Comment:
I don't think you need the explanation about not deleting the comments in this doc block. Instead I'd put this at the call site in command_runner